redisdb/README.md

```diff
@@ -302,8 +302,6 @@ See [service_checks.json][24] for a list of service checks provided by this inte
 
 ## Troubleshooting
 
-- [Redis Integration Error: "unknown command 'CONFIG'"][25]
```

Comment:
Do you happen to know where this page went/why it's gone?

Reply:
This page was deleted, further context is available within DOCS-3484